src/components/movieList/movieList.js

```diff
@@ -0,0 +1,80 @@
+import s from './movieList.scss';
+import MovieCard from '../movieCard/movieCard';
+import Button from '../button/button';
+import SelectButton from '../selectButton/selectButton';
+
+const MovieList = () => (
+  <main className={s.movieList}>
+    <header className={s.movieList__header}>
+      <nav>
+        <Button
```

Comment:
one of the possible variant to implement it
putt all data to array of objects
then just map it
what do you think? It's not required, just a suggestion